lib/gcp_auto_attach.py

```diff
@@ -18,7 +19,9 @@ def gcp_auto_attach(cfg: config.UAConfig) -> None:
         # If we are not running on GCP cloud, we shouldn't run this
         # job anymore
         LOG.info("Disabling gcp_auto_attach job. Not running on GCP instance")
-        cfg.gcp_auto_attach_timer = 0
+        cfg.delete_cache_key("marker-license-check")
+        # HEY REVIEWERS LOOK HERE this next line seems scary to have here
+        util.subp(["systemctl", "restart", "ua-license-check.timer"])
```

Comment:
If we are deleting the marker file, and our systemd PathExists declaration works to properly enable whenever that marker file appears, then I think what we need here is `systemctl stop ua-license-check.timer`.

Reply:
good point!